fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/physical/PhysicalScanOperator.java

```diff
@@ -63,6 +63,14 @@ public List<ColumnRefOperator> getOutputColumns() {
         return outputColumns;
     }
 
+    public List<ColumnRefOperator> getRealOutputColumns() {
+        if (projection != null) {
+            return projection.getOutputColumns();
+        } else {
+            return outputColumns;
+        }
+    }
```

Comment:
why don't modify outputColumns？

Reply:
I modified it in another PR, and many test cases failed.